draft-trammell-taps-interface.md

```diff
@@ -285,94 +288,69 @@ localSpecifier.withStunServer(address, port, credentials)
 
 The transport services API will resolve names internally, when the Initiate(),
 Listen(), or Rendezvous() method is called establish a connection.
 The API does not need the application to resolve names, and premature name
 resolution can damage performance by limiting the scope for alternate path
 discovery during connection establishment.
-The Resolve() method is, however, provided to resolve a LocalEndpoint or a
-RemoteEndpoint in cases where this is required, for example with some NAT
+The Resolve() method is, however, provided to resolve a Local Endpoint or a
+Remote Endpoint in cases where this is required, for example with some NAT
 traversal protocols (see {{rendezvous}}).
 
-
 \[NOTE: the API needs MUST be explicit about when name resolution occurs,
         since the act of resolving a name leaks information, and there
         may be security implications if this happens unexpectedly.]
 
 ## Specifying Transport Parameters {#transport-params}
 
 A Preconnection object holds parameters reflecting the application's
-requirements and preferences for the transport.  These include Transport
-Preferences (towards protocol selection and path selection) as well as
-Application Intents (hints to the transport system what to optimize for)
-and Protocol Properties (to configure transport protocols).
-
-All Transport Parameters are organized within a single name space, that is
-shared with Send Parameters (see {{send-params}}). While Application
-Intents and Protocol Properties take parameter specific values, Transport
-Preferences use one of five fixed levels (see {{transport-prefs}}).
-
-Some parameters express strict requirements that the application relies on,
-while others are hints of what transport features would be helpful
-for the application. For example, if an application asks for reliable data
-transfer, choosing a transport protocol such as UDP, which does not have this
-feature, will break the application's functionality. On the other hand, the
-option to not require checksums when receiving an individual Content can help
-optimize for low latency, but if not present, it will most likely not break the
-fundamental assumptions of the application.
-Moreover, there can be conflicts between parameters set by
-the application: If multiple features are requested which are offered
-by different protocols, it may not be possible to satisfy all requirements.
-Consequently, a transport system must prioritize Transport Parameters and
-consider the relevant trade-offs, see also {{?TAPS-MINSET=I-D.ietf-taps-minset}}.
-
-Default preference levels and possible combinations of Transport Parameters and
-preference levels are specified in {{appendix-preferences}}.
-
-
-
-### Transport Preferences {#transport-prefs}
-
-Transport Preferences drive protocol selection and path selection on
-connection establishment. Since there could be paths over which some transport
+requirements and preferences for the transport. These include protocol and path
```

Comment:
Capitalize Protocol and Path Selection here, since they are terms from the architecture?